src/json/members/august.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "August Fu",
+    "pennkey": "yuntongf",
+    "major": "Computer Science",
+    "school": "Engineering",
+    "bio": "Salt air, and the rust on your door\n I never needed anything more\n Whispers of \"Are you sure?\" \nNever have I ever before\"",
+    "hometown": "Tianjin China",
+    "team": "Penn Courses",
+    "roles": ["Frontend Engineer"],
+    "photo": "https://media.licdn.com/dms/image/D4E03AQHKm_uMFR55Zw/profile-displayphoto-shrink_800_800/0/1686604042664?e=1701302400&v=beta&t=0S1WNZkHfjgYDwf2ky2BBLLgFI0tqx0ww1Jkptk0h2Y",
```

Comment:
It's easy for this type of photo to no longer have a valid URL after a certain period of time. Rather than right clicking on LinkedIn profile to copy image URL, I would recommend using a hosting site like Imgur.